datadog_checks_base/datadog_checks/base/checks/win/winpdh.py

```diff
@@ -72,9 +73,17 @@ def __del__(self):
     def is_single_instance(self):
         return self._is_single_instance
 
+    @property
+    def localized_class_name(self):
```

Comment:
```suggestion
    def class_name(self):
```
It can also be the english one depending on the circumstances so I find the name misleading